money_notebook_v141.py

```diff
@@ -417,6 +497,17 @@ def total_cal(s,e):
             elif len(uinput_split) == 2:
                 total_cal(uinput_split[1],uinput_split[1])
         else:print("【无效输入（错误代码14）】")
-        show_m = False
-    elif uinput == "f":break
+    elif re.search("\+", uinput) and len(uinput) == 1:
+        edit_plan(uinput)
+        show_menu()
+    elif not re.search("\D", uinput[2:]) and re.search("^- ", uinput):
+        edit_plan(uinput[2:])
+        show_menu()
+    elif uinput == "f":
+        # backup file daily everytime the program closes
+        today = datetime.date.today().strftime("%Y-%m-%d")
+        file_name = "memo_backup" + today + ".txt"

```

Comment:
Move "memo_backup" to the top of the file, for example
~~~python
BACKUP_FOLDER = "memo_backup"
file_name = BACKUP_FOLDER  + today + ".txt"
~~~